test/unit/sql/select.test.js

```diff
@@ -371,6 +371,49 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
         default: 'SELECT [User].[name], [User].[age], [Posts].[id] AS [Posts.id], [Posts].[title] AS [Posts.title] FROM [User] AS [User] LEFT OUTER JOIN [Post] AS [Posts] ON [User].[id] = [Posts].[user_id];'
       });
     });
+
+    it('include (subQuery alias)', function () {
+      var User = Support.sequelize.define('User', {
```

Comment:
I can't use 'const' for branches of v3, as tests are run on nodejs version 0.10 and 0.12. In accordance with the website http://node.green/ full const support starts in version 6.4.0.